.changeset/stale-brooms-fry.md

```diff
@@ -0,0 +1,51 @@
+---
+'@backstage/create-app': patch
+---
+
+You can now add `SidebarGroups` to the current `Sidebar`. This will not affect how the current sidebar is displayed, but allows a customization on how the `MobileSidebar` on smaller screens will look like. A `SidebarGroup` will be displayed with the given icon in the `MobileSidebar`.
```

Comment:
Super NIT. `SidebarGroups` should be `SidebarGroup`s right? It's also in the other changeset. 

Overall I think this is goood to go 👍 🤞